native/src/utils/renderHtml.ts

```diff
@@ -71,34 +84,143 @@ const renderJS = (cacheDictionary: Record<string, string>) => `
     }
   })();
 
-  (function() {
-    var container = document.getElementById('measure-container')
+  (function addWebviewHeightListeners() {
+    const container = document.getElementById('measure-container')
 
     function adjustHeight () {
-      container.setAttribute('style', 'padding: 1px 0;'); // Used for measuring collapsed vertical margins
+      container.setAttribute('style', 'padding: 1px 0;') // Used for measuring collapsed vertical margins
 
       if (!window.ReactNativeWebView) {
-        return window.setTimeout(adjustHeight, 100);
+        return window.setTimeout(adjustHeight, 100)
       }
 
-      var height = container.getBoundingClientRect().height - 2
-      window.ReactNativeWebView.postMessage(JSON.stringify({ type: '${HEIGHT_MESSAGE_TYPE}', height: height }));
-      container.setAttribute('style', '');
+      const height = container.getBoundingClientRect().height - 2
+      window.ReactNativeWebView.postMessage(JSON.stringify({ type: '${HEIGHT_MESSAGE_TYPE}', height: height }))
+      container.setAttribute('style', '')
     }
 
-    window.addEventListener('load', adjustHeight);
-    window.addEventListener('resize', adjustHeight);
-    var details = document.querySelectorAll("details")
-    details.forEach(detail => detail.addEventListener("toggle", adjustHeight))
+    window.addEventListener('load', adjustHeight)
+    window.addEventListener('resize', adjustHeight)
+    const details = document.querySelectorAll('details')
+    details.forEach(detail => detail.addEventListener('toggle', adjustHeight))
   })();
+
+  (function handleIframes() {
```

Comment:
I haven't checked for alternatives yet but this feels super hard to read/review and looks pretty error prone. Maybe we should discuss this a little more. Or are we sure that this is the best option?

Reply:
dunno what other possibilities we have inside this template literals